README.md

```diff
@@ -1,3 +1,15 @@
 # llvm-remote-index
 
 [![Index LLVM project](https://github.com/clangd/llvm-remote-index/workflows/Index%20LLVM%20project/badge.svg)](https://github.com/clangd/llvm-remote-index/actions)
+
+# Creating a docker image
+
+Issuing `bash deployment/create_docker_image.sh` will create a new container for
```

Comment:
nit: Running `bash ...`? Never seen "issuing" in this context, might be unexpected for others, too :)